glib/src/object.rs

```diff
@@ -1227,50 +1239,108 @@ pub trait ObjectExt: ObjectType {
+    ///
+    /// `None` is returned if the object is not a subclass of `T`.
     #[doc(alias = "get_class_of")]
     fn class_of<T: IsClass>(&self) -> Option<&Class<T>>;
+
+    /// Returns the interface `T` of the object.
+    ///
+    /// `None` is returned if the object does not implement the interface `T`.
     #[doc(alias = "get_interface")]
     fn interface<T: IsInterface>(&self) -> Option<InterfaceRef<T>>;
 
+    /// Sets the property `property_name` of the object to value `value`.
+    ///
+    /// This fails if the property does not exist, or if the type of the property is different than
+    /// the provided value, or if the property is not writable.
     #[doc(alias = "g_object_set_property")]
     fn set_property<'a, N: Into<&'a str>, V: ToValue>(
         &self,
         property_name: N,
         value: V,
     ) -> Result<(), BoolError>;
+
+    /// Sets the property `property_name` of the object to value `value`.
+    ///
+    /// This fails if the property does not exist, or if the type of the property is different than
+    /// the provided value, or if the property is not writable.
     fn set_property_from_value<'a, N: Into<&'a str>>(
         &self,
         property_name: N,
         value: &Value,
     ) -> Result<(), BoolError>;
+
+    /// Sets multiple properties of the object at once.
+    ///
+    /// This does not set any properties if one or more properties don't exist, or values of the
+    /// wrong type are provided, or if any of the properties is not writable.
+    #[doc(alias = "g_object_set")]
     fn set_properties(&self, property_values: &[(&str, &dyn ToValue)]) -> Result<(), BoolError>;
+
+    /// Sets multiple properties of the object at once.
+    ///
+    /// This does not set any properties if one or more properties don't exist, or values of the
+    /// wrong type are provided, or if any of the properties is not writable.
     fn set_properties_from_value(&self, property_values: &[(&str, Value)])
         -> Result<(), BoolError>;
+
+    /// Gets the property `property_name` of the object.
+    ///
+    /// This fails if the property does not exist or is not writable.
     #[doc(alias = "get_property")]
     #[doc(alias = "g_object_get_property")]
     fn property<'a, N: Into<&'a str>>(&self, property_name: N) -> Result<Value, BoolError>;
+
+    /// Check if the object has a property `property_name` of the given `type_`.
+    ///
+    /// If no type is provided then only the existence of the property is checked.
     fn has_property<'a, N: Into<&'a str>>(&self, property_name: N, type_: Option<Type>) -> bool;
+
+    /// Get the type of the property `property_name` of this object.
+    ///
+    /// This returns `None` if the property does not exist.
     #[doc(alias = "get_property_type")]
     fn property_type<'a, N: Into<&'a str>>(&self, property_name: N) -> Option<Type>;
+
+    /// Get the [`ParamSpec`](crate::ParamSpec) of the property `property_name` of this object.
```

Comment:
I don't think you need to pass the URL. Did you try:

```suggestion
    /// Get the [`ParamSpec`] of the property `property_name` of this object.
```

?

Reply:
Then it prints `crate::ParamSpec` in the docs, that's ugly.

Reply:
Ah no ok, now I know why: you need to specify `crate::` to make the link work, which is why you created a link to improve the rendering. Makes sense!